otcextensions/sdk/modelartsv2/v2/dataset_sync.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+from openstack import resource
+
+
+class DatasetSync(resource.Resource):
+    base_path = "/datasets/%(datasetId)s/sync-data"
+
+    allow_create = True
+    allow_fetch = True
+
+    # Resource Key
+    #: Dataset ID.
+    datasetId = resource.URI("datasetId")
```

Comment:
dataset_id

Reply:
datasetId - resource.URI # for base_path
dataset_id - response.Body

Reply:
updated datasetId to uri_dataset_id